src/loom-provider.ts

```diff
@@ -171,8 +178,10 @@ export class LoomProvider {
         return createDefaultTxMiddleware(client, privateKey)
       }
     }
+    this._addDefaultMethods()
```

Comment:
How is this supposed to work exactly? When the client is setup to use the `/eth` endpoint the `eth_XXX` methods should be handled by `_ethCallSupportedMethod` which skips the protobuf marshalling & unmarshalling. This change here prevents that from happening, and send protobufs to the `/eth` endpoint which doesn't know what to do with them.

Reply:
My bad this one will be fixed